storey/aggregations.py

```diff
@@ -561,7 +598,8 @@ def get_aggregation_for_aggregation(self):
             return 'sum'
         return self.aggregation
 
-    def get_features(self, timestamp):
+    def get_features(self, timestamp, get_hidden=False):
```

Comment:
we probably need to also request specific windows, because if a user queries for `sum_3h`, `avg_1h`, `stdvar_2h`:
when avg requests sum for virtual aggregates it will get 1h&2h which probably blow things up